src/architecture/utilities/keplerianOrbit.h

```diff
@@ -30,7 +30,7 @@
 class KeplerianOrbit {
 public:
     KeplerianOrbit();
-    KeplerianOrbit(classicElements oe, GravBodyData* planet);
```

Comment:
on line 23, remove the include of gravityEffector.h

Reply:
Thanks for the catch! Done.